Automattic/wp-calypso
# Add-Ons: Add Jetpack AI card to Add-Ons page

Related to p1688046982126439-slack-C054LN8RNVA
p1HpG7-nrG-p2

## Proposed Changes

* Add Jetpack AI card to Add-ons page(wordpress.com/add-ons/yoursite.com)
* It should only be available on a dev environment, the card is hidden behind the `jetpack/ai-assistant-request-limit` feature flag

## Testing Instructions



* Checkout this branch and follow the [getting started instructions to have Calypso running locally](https://github.com/Automattic/wp-calypso#getting-started)
* Once it's running locally, on the menu, go to "Upgrade > Add-ons"
* Confirm the Jetpack AI card is showing:

<img width="600" alt="Screen Shot 2023-07-12 at 18 58 50" src="https://github.com/Automattic/wp-calypso/assets/6760046/1986de26-26c4-44e1-bd67-121d25a09a6e">

* Add the following parameter to the end of the URL and load the page again; it will disable the feature flag, so we can check if it's working as expected: `?flags=-jetpack/ai-assistant-request-limit` (notice the `-` signal, it's necessary)
   * Example: `http://calypso.localhost:3000/add-ons/my-test-site.wordpress.com?flags=-jetpack/ai-assistant-request-limit`
* In this case, the add-on should not show up:

<img width="600" alt="Screen Shot 2023-07-12 at 19 11 46" src="https://github.com/Automattic/wp-calypso/assets/6760046/828abd5a-abc4-4bb7-831f-82dd8ca8145c">

## Pre-merge Checklist
